communities/genome/lab/annotation.yml

```diff
@@ -357,4 +393,5 @@ tabs:
             Any user of Galaxy Australia can request support through an online form.
           </p>
         button_md: Request support
-        button_link: /request/support
+        button_link: https://site.usegalaxy.org.au/request/support
+        button_tip: Click to request
```

Comment:
Users know that buttons should be clicked, just remove this tooltip unless it adds value, same with the above.

```yaml
button_tip: ""
```